languages/rust/reference/exercise-concepts/poker.md

```diff
@@ -0,0 +1,35 @@
+# Concepts Required for the Poker exercise
+
+There are many ways to potentially solve an exercise, but I'll be looking at the [canonical example](https://github.com/exercism/rust/blob/c0afe3855e0e71dc5429aa97657e074aa90a8e11/exercises/poker/example.rs) as my primary reference for what concepts are used.
+
+When referring to the list of concepts, we mean [this list](https://github.com/exercism/v3/blob/4f164e3751202087ff40c8321238d4c283a90fd0/languages/rust/reference/README.md)

```

Comment:
```suggestion
When referring to the list of concepts, we mean [this list](https://github.com/exercism/v3/blob/master/languages/rust/reference/README.md)

```
I figured targeting master is most helpful since now since we'll be updating it as we go.